cmd/clusterlink/cmd/deploy/deploy_peer.go

```diff
@@ -103,6 +116,14 @@ func (o *PeerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.Uint16Var(&o.IngressPort, "ingress-port", apis.DefaultExternalPort,
 		"Represents the ingress port. By default it is set to 443 for LoadBalancer"+
 			" and a random port in range (30000 to 32767) for NodePort.\nThis option is only valid if --autostart is set.")
+	fs.StringVar(&o.DataplaneType, "dataplane", platform.DataplaneTypeEnvoy,
+		"Type of dataplane, Supported values: \"envoy\" (default), \"go\"")
+	fs.Uint16Var(&o.DataplaneReplicas, "dataplane-replicas", 1, "Number of dataplanes.")
+	fs.StringVar(&o.LogLevel, "log-level", "info",
+		"The log level. One of fatal, error, warn, info, debug.")
+	fs.BoolVar(&o.DryRun, "dry-run", false,
```

Comment:
this parameter should be in the documentation (replaces `autostart`?)

Reply:
check in the documentation if it is not clear enough,
without any flag: just the operator start
--autostart deploy also the instance YAML to the operator so the CLusterLink components start.
--dryrun doesn't do anything just create the YAMLs (doesn't deploy the operator).

Reply:
since dry-run and auto-start are mutually exclusive (ie I should not define both) maybe combine into a single parameter?
`--components`: `operator` (default), `all` (operator and clusterlink), `none` (generate YAMLs only)
Not sure if `components` is the right flag name (maybe `deploy`?), but you get the idea, I hope.

Reply:
Done, please check the commit https://github.com/clusterlink-net/clusterlink/pull/507/commits/0c0f655af429a5d72f1262bb5428ea0de5c50736